models/base.py

```diff
@@ -120,12 +120,15 @@ def __init__(self, maker: ILayerMaker, layer_dict: LayerDictRaw):
     self.maker = maker
     self.layer_dict = layer_dict
 
-  def mark_as_loss(self):
+  def mark_as_loss(self, loss_scale=1.0):
     """
     Mark this as a loss.
+    :param loss_scale: scale factor for loss (1.0 by default)
```

Comment:
Make an empty line after the description before the first `:param` (consistency).

Or maybe even better: Just remove this line. It doesn't really add any useful information.
```suggestion
```